crates/holochain/src/conductor/manager/mod.rs

```diff
@@ -20,6 +41,165 @@ use tracing::*;
 
 use super::ConductorHandle;
 
+/// The main interface for interacting with a task manager.
+/// Contains functions for adding tasks to groups, stopping task groups,
+/// and shutting down the task manager.
+#[derive(Clone)]
+pub struct TaskManagerClient {
```

Comment:
I just moved TaskManagerClient to the top of the file since that is the actual public interface. And added a few more docs.